tips/TIP-0048/openapi3-core.yaml

```diff
@@ -0,0 +1,3805 @@
+      properties:
+        slot:
+          type: integer
+          description: The slot index for which the congestion estimate is provided.
+        ready:
+          type: boolean
+          description: >
+            Indicates if a node is ready to issue a block in a current congestion or should wait.
+        referenceManaCost:
+          type: string
+          description: The cost in mana for issuing a block in a current congestion estimated based on RMC and slot index.
+        blockIssuanceCredits:
+          type: string
+          description: The Block Issuance Credits of the requested account.
+      required:
+        - slot
+        - ready
+        - referenceManaCost
+        - blockIssuanceCredits
+
+
+    Commitment:
+      description: An object embedded to a block, containing a summary of a slot.
+      properties:
+        protocolVersion:
+          type: integer
+          description: The version of the protocol running.
+        slot:
+          type: integer
+          description: The slot index of the commitment.
+        previousCommitmentId:
+          type: string
+          description: The commitment identifier of the previous slot.
+        rootsId:
+          type: string
+          description: The digest of multiple merkle roots within this slot.
+        cumulativeWeight:
+          type: string
+          description:
+            The sum of previous slot commitment cumulative weight and 
+            weight of issuers of accepted blocks within <code>index</code> minus <code>Maximum Committable Age</code>. It indicates the weight of the chain at <code>index</code> minus <code>Maximum Committable Age</code> committed at <code>index</code> slot.
+        referenceManaCost:
+          type: string
+          description: This field provides the Reference Mana Cost (RMC) to be used in the slot with index Index + Max Committable Age.
+      required:
+        - protocolVersion
+        - slot
+        - previousCommitmentId
+        - rootsId
+        - cumulativeWeight
+        - referenceManaCost
+
+    SubmitBlockRequest:
+      description: Submits a block to the node.
+      properties:
+        allOf:
+          $ref: '#/components/schemas/Block'
+
+    SubmitBlockResponse:
+      description: Returns the block identifier of the submitted block.
+      properties:
+        blockId:
+          type: string
+          description: The block identifier of the submitted block. Hex-encoded with 0x prefix.
+      required:
+        - blockId
+
+    BlockFullResponse:
+      description: Returns the block structure and the metadata of a given block.
+      properties:
+        block:
+          $ref: '#/components/schemas/Block'
+          description: The requested block structure.
+        metadata:
+          $ref: '#/components/schemas/BlockMetadataResponse'
+          description: The metadata of the requested block.
+      required:
+        - block
+        - metadata
```

Comment:
Can this be renamed to `BlockWithMetadataResponse`? Would be consistent with `OutputWithMetadataResponse`.